tests/commands/create/test_create_app.py

```diff
@@ -81,6 +93,38 @@ def test_create_existing_app_no_overwrite_default(tracking_create_command):
 
     tracking_create_command.create_app(tracking_create_command.apps['first'])
 
+    # Input was required by the user
+    assert tracking_create_command.input.prompts == [
+        'Application first already exists; overwrite? [y,N]'
+    ]
+
+    # And no actions were necessary
+    assert tracking_create_command.actions == []
+
+    # Original content still exists
+    assert (bundle_path / 'original').exists()
+
+    # New app content has not been created
+    assert not (bundle_path / 'new').exists()
+
+
+def test_create_existing_app_input_disabled(tracking_create_command):
+    "If input is disabled, fallback to default without get input from user"
+    # Answer '' (i.e., just press return) when asked
+    tracking_create_command.input.disable()
+    tracking_create_command.input.set_values('y')
```

Comment:
Is this necessary? My fear is that it might inadvertently mask a situation where input has been disabled, but the test harness is mocking a request for input. Would it be better to put the tracking command into an "explosive" state so any request that actually requests input causes an error?